Update mac.md

oddlittlebird opened the pull request:
Reviewed and edited macOS install instructions.

**What this PR does / why we need it**: Adds clarity to macOS installation instructions.

**Which issue(s) this PR fixes**: Part of #19893

Fixes #17183

**Special notes for your reviewer**: Please refer to questions in comments.

oddlittlebird committed:
Update mac.md

Reviewed and edited macOS install instructions.

oddlittlebird commented on docs/sources/installation/mac.md:
```diff
@@ -9,49 +9,50 @@ weight = 500
+# Install on macOS
 
-This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
+This page provides instructions to help you install Grafana on macOS. You can either install using Homebrew or a binary tar file.
```
Is the binary tar file still an option? I do not see it on the [Download Grafana](https://grafana.com/grafana/download?platform=mac) page.

oddlittlebird replied:
@DanCech ?

oddlittlebird replied:
*pokes* @DanCech

oddlittlebird commented on docs/sources/installation/mac.md:
```diff
@@ -9,49 +9,50 @@ weight = 500
-To start grafana look at the command printed after the homebrew install completes.
-
-To upgrade use the reinstall command
+You can also install the latest unstable Grafana version from Git:
```
Do we really want to tell them how to do this? Should this be in the Developer's Guide instead?

oddlittlebird replied:
@marcusolsson ?

marcusolsson replied:
Any mention of building Grafana from source should link to the Developer Guide, but in this case I don't think we need to tell them how to install Grafana from master using brew at all.

oddlittlebird replied:
Removed all references to building Grafana from master in this doc, moved configuration information to configuration.md.

oddlittlebird commented on docs/sources/installation/mac.md:
```diff
@@ -9,49 +9,50 @@ weight = 500
 ### Start Grafana
 
-To start Grafana using homebrew services first make sure homebrew/services is installed.
+To start Grafana using Homebrew services: 
```
Is Homebrew Services a separate installation from Homebrew? Would the user ever have Homebrew installed (and thus be able to perform the earlier installation) without having the services installed?

oddlittlebird replied:
@marcusolsson or @torkelo ?

marcusolsson replied:
Homebrew services is installed on the first use, i.e. upon running `brew services`.

oddlittlebird commented on docs/sources/installation/mac.md:
```diff
@@ -63,19 +64,19 @@ Then start Grafana using:
+Refer to [Getting started](docs\sources\guides\getting_started.md) for instructions on logging in for the first time and adding a data source.
 
-## Install from binary tar file
+## Install from binary .tar file
```
See earlier question: Is this section relevant anymore? The link to grafana.com/get leads to either the Cloud install or back to the Downloads page, where there is no .tar.gz file option.

oddlittlebird replied:
@DanCech ?

oddlittlebird replied:
@DanCech, need an answer on this.

oddlittlebird replied:
I deleted the binary .tar since it is not available on the Mac page.

oddlittlebird committed:
Updated mac.md and configuration.md

Removed developer build instructions from mac.md and moved configuration information from mac.md to configuration.md.

oddlittlebird committed:
Update mac.md

Updated to reflect the new fields on the Downloads page

oddlittlebird committed:
Update mac.md

srclosson commented on docs/sources/installation/mac.md:
````diff
@@ -8,74 +8,43 @@ parent = "installation"
+brew reinstall grafana
 ```
-
-Default login and password `admin`/ `admin`
````
Might be nitpicky, but did you intend to have the whitespace after but not before?

oddlittlebird replied:
Not sure what you are seeing.

oddlittlebird replied:
Current mac.md ends at line 51.

srclosson commented on docs/sources/installation/mac.md:
```diff
@@ -8,74 +8,43 @@ parent = "installation"
-
-## Install from binary tar file
-
-Download [the latest `.tar.gz` file](https://grafana.com/get) and
```
Did we want to document the process for setting up Grafana as a service? If so, even a link here to some other documentation would be very nice.

oddlittlebird replied:
Actually, I removed that section in the last commit. You currently cannot get the .tar.gz file on the Mac page.

DanCech replied:
See comment above, I'm not sure whether there is a reason that we don't offer people the option to download the osx binaries directly (if they are published for a release).

srclosson commented on docs/sources/installation/mac.md:
```diff
@@ -8,74 +8,43 @@ parent = "installation"
+   * The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
+2. Select an **Edition**.
+   * **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
+   * **Open Source** - Functionally identical to the enterprise version, but you will need to download the enterprise version if you want enterprise features.
```
Would a link here to highlight what the difference between the enterprise features vs. open source are?

oddlittlebird replied:
Currently does not exist. I am trying to get someone to write the page for the website.

DanCech replied:
My understanding is that we don't currently have a homebrew setup for grafana enterprise.

The current enterprise release also does not appear to have osx binaries available, but my understanding is that someone was working on fixing that so we would build osx binaries for future enterprise releases.

For the current stable release there is a tarball published with the oss osx binaries, but the downloads page explicitly does not offer them for download and instead directs folks to use homebrew.  I don't know why that is, but it would be fairly easy to give people the option to download the binaries directly.

oddlittlebird replied:
Nice catch on Enterprise. Updated.

ivanahuckova reviewed (commented):
Hi @oddlittlebird.  I am slightly confused regarding customisation section. I am on **macOS**, but I am configuring files the same way,  as it it described in **Windows** section, and it works for me. Based on this PR, I have checked, that I have  grafana.ini file here -`/usr/local/etc/grafana/grafana.ini`, but I didn't know about it up until now. I always thought that `defaults.ini` and my `custom.ini` are only configuration files. 

Other than that, everything looks great! 🙂

oddlittlebird commented:
@ivanahuckova, that confusion is why I am removing all the extra content about configuration, logging in, and so on from the installation topics.

oddlittlebird commented:
Any other changes needed, or can we publish this?

oddlittlebird committed:
Update mac.md

oddlittlebird commented:
Any other changes needed, or can we approve and publish this?

ivanahuckova reviewed (approved):
From my point of view, everything looks clear.  No changes necessary. Good job @oddlittlebird! 🔥
